controllers/users_controller.js

```diff
@@ -4,4 +4,58 @@ const router = express.Router();
 const connection = require("../config/connection");
 
 // export the router at the end of your file.
-module.exports = router;
\ No newline at end of file
+module.exports = router;
+
+
+app.get('/', function (req, res) {
```

Comment:
app.get should be router.get because we are using multiple controllers (events, users and interests) I will link you to my HW files so you can see how I did it there! This looks great though, awesome job! :D